src/org/zaproxy/zap/extension/spider/SpiderPanel.java

```diff
@@ -160,6 +163,9 @@
 	/** The found count value label. */
 	private JLabel foundCountValueLabel;
 	
+	private static ZapTableExportButton exportUrlsButton = null;
+	private static ZapTableExportButton exportMessagesButton = null;

```

Comment:
I'd suggest changing `ZapTableExportButton` to allow to set a table and use just one button.